### ISSUE TITLE
CIFTI JSON sidecar is unindented

### ISSUE THREAD

**Issue by effigies** [poldracklab/fmriprep#2002]
Currently I get:

```JSON
{"space": "HCP grayordinates", "surface": "fsLR", "volume": "MNI152NLin6Asym", "surface_density": "32k", "grayordinates": "91k"}
```

It would improve readability to have these indented.

Extremely low priority.

**Comment by mgxd** [comment]
we can fit this in to the niworkflows 1.1.8 release

**Comment by effigies** [comment]
Okay. Marked it 20.0.1 then. Feel free to push off for any reason.

**Comment by effigies** [comment]
Closed by 2590fed406a77874f6ab5dc498d246bc0f0db4bc.
